ts/data/case.ts

```diff
@@ -187,7 +187,24 @@ export const caseDefinition: TableDefinition<Case, Test> = {
     }
     return null;
   },
-  staticActions: [legendAction],
+  staticActions: [
+    legendAction,
+    {
+      title: "TAT Report",
+      handler: (filters: { key: string; value: string }[]) => {
+        const currentFilters: { [key: string]: any } = {};
+        for (const filter of filters) {
+          if (filter.value !== undefined && filter.value !== null) {
+            currentFilters[filter.key] = filter.value;
+          }
```

Comment:
This won't allow multiple filters of the same type - a second one would overwrite the first (e.g. if you filter for 2 different projects)